guides/micronaut-openapi-generator-server/micronaut-openapi-generator-server.adoc

```diff
@@ -0,0 +1,550 @@
+
+Now we will generate server API files from our definition. The generated server code will be in Java and will use the Micronaut features for client-server communication.
+
+:exclude-for-build: maven
+
+Open your `build.gradle` file an apply the `micronaut-openapi` plugin:
+
+
+[source, groovy]
+.build.gradle
+----
+plugins {
+  id 'io.micronaut.openapi' version '...'
+  ....
+}
+----
+
+And configure your build to generate a client:
+
+[source, groovy]
+.build.gradle
+----
+micronaut {
+  ...
+  openapi {
+    server(file("src/main/resources/library-definition.yml")) { // <1>
+      apiPackageName = "example.micronaut.api"
+      modelPackageName = "example.micronaut.model"
+      invokerPackage = "example.micronaut.invoker"
+      useReactive = false                                       // <2>
+      useAuth = true                                            // <3>
+    }
+  }
+}
+----
+<1> Loads the service definition in order to generate a server
+<2> Configures the generator to not generate reactive code
+<3> Configures the generator to support authentication
+
+The server code will be generated in your build directory but automatically added as a source set.
+Therefore, you can for example run `./gradlew compileJava --console=verbose` and see that the sources are generated and compiled:
+
+[source]
+----
+> Task :generateServerOpenApiApis
+...
+> Task :generateServerOpenApiModels
+...
+> Task :compileJava
+----
+
+TIP: The Micronaut OpenAPI generator supports a large number of parameters. Please refer to the https://micronaut-projects.github.io/micronaut-gradle-plugin/snapshot/#_generating_a_server[Micronaut OpenAPI Gradle plugin documentation] for all possible options.
+
+After generation finishes, you should see the following directory structure under your `build/generated/openapi` directory:
+
+[source]
+----
+build/generated/openapi
+├── generateServerOpenApiApis
+│   └── src
+│       └── main
+│           ├── java
+│           │   └── example
+│           │       └── micronaut
+│           │           ├── api
+│           │           │   └── BooksApi.java       # <1>
+│           │           └── invoker
+│           │               └── annotation
+│           │                   └── HardNullable.java
+├── generateServerOpenApiModels
+│   └── src
+│       └── main
+│           └── java
+│               └── example
+│                   └── micronaut
+│                       └── model                              # <2>
+│                           ├── BookAvailability.java
+│                           └── BookInfo.java
+----
+<1> The api package will contain generated interfaces for our API that we will need to implement in our controller
+<2> It also generates model classes which are required by the service
+
+:exclude-for-build:
+
+:exclude-for-build: gradle
+
+Enable the Micronaut OpenAPI server generation in your `pom.xml` file, configure the location to the specification file and package names:
+
+[source, xml]
```

Comment:
This ends up in the Gradle guide still

Reply:
Fixed in https://github.com/micronaut-projects/micronaut-guides/pull/1298/commits/7ec014397a5faf47a675e64a4cbcfef6c1446915